docs/concepts/ipld.md

```diff
@@ -11,4 +11,94 @@ related:
 
 # The InterPlanetary Linked Data (IPLD) model
 
-<ContentStatus />
+The Interplanetary Linked Data (IPLD) model is a set of [specifications](https://github.com/ipld/specs) that unifies all data models that link data with hashes. This allows for interoperable protocols. For example, you can reference a github commit to a bitcoin transaction.
+
+IPFS, bitcoin and other applications use hash-linked data structures (Merkle trees) to verify data integrity. However, each protocol has a slightly different format. IPLD's goal is to provide a hash-chain format for distributed data structures.
+
+IPLD concerns itself with the data layer of the distributed web. It beings with how data is encoded and decoded. IPLD's [Data Model](https://github.com/ipld/specs) includes standard types such as booleans, integers, lists etc. which are easily representable by common programming languages.
+
+## Where Does IPLD Fit In?
+
+IPLD acts as a layer between networking protocols and higher level applications. This enables developers to build decentralized applications without having to worry about lower level changes.
+
+![](https://gateway.ipfs.io/ipfs/QmXgrfpCcSFfXnXqSz6G3V9E21pTZdBmVdsBQCkr86kHXP)
+
+## IPLD Layer Model
+
+The IPLD specifications are divided into [layers](https://github.com/ipld/specs#ipld-layer-model):
+
+0. Block Layer: The block layer contains the CID which is used to self describe the multibase-prefix, version, content type and content address. This takes the guessing out of encoding/decoding.
+1. Data Model Layer: This layer describes what is representable in IPLD (lists, booleans, maps etc). Here are all of the [kinds](https://github.com/ipld/specs/blob/master/data-model-layer/data-model.md#kinds).
+1. Schema Layer: The schema layer uses the data model layer to build more complex layouts.
+
+Below is the simplified hierarchy:
+
+```
+┌──────────────────────────────────────────────────┐
+│                                                  │
+│                   2. Schema Layer                │
+│ (advanced types for multi-block data structures) │
+│                                                  │
+├──────────────────────────────────────────────────┤
+│                                                  │
+│                 1. Data Model Layer              │
+│  (basic types for single-block data structures)  │
+│                                                  │
+├──────────────────────────────────────────────────┤
+│                                                  │
+│                   0. Block Layer                 │
+│               (cid, data, codec)                 │
+│                                                  │
+└──────────────────────────────────────────────────┘
+```
+
+## Path Traversal Example
+
+This example will create a simple link from a node to a git commit. The example will use the cli, but this can be done in many ways ([javascript](https://github.com/ipfs/js-ipfs-http-client)).
```

Comment:
```suggestion
This example will create a simple link from a node to a Git commit. The example will use the CLI, but this can be done in many other ways (for example, using [JavaScript](https://github.com/ipfs/js-ipfs-http-client)).
```